packages/rest-generator/src/generator.ts

```diff
@@ -128,14 +128,12 @@ async function generateFilesUsingOpenAPI(
 
   logger.info(`Argument for openapi generator ${generationArguments}`);
 
-  try {
-    const response = await execa.sync('npx', generationArguments);
-    if (response !== undefined) {
-      logger.info(`Generated the client ${response.stdout}`);
-    }
-  } catch (err) {
-    logger.error('In exception block');
-    logger.error(err);
+  const response = await execa.sync('npx', generationArguments);
+  if (response.stderr !== '') {
+    throw new Error(response.stderr);
+  }
+  if (response !== undefined) {
```

Comment:
here is the null check, then response is nullable.
If response is null, then line 132 will fail

Reply:
yes, the undefined check shoud be first.